### PR TITLE
Fix `websocket_url` on the minimal WebSocket client example

### PR DESCRIPTION
**Pull request by AvicennaJr**
The current URL on the "Minimal Client Example" is no longer functional and needs to be updated to a working one.

### CODE REVIEW

File: tutorials/networking/websocket.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ This example will show you how to create a WebSocket connection to a remote serv
     extends Node
 
     # The URL we will connect to
-    export var websocket_url = "ws://echo.websocket.org"
+    export var websocket_url = ""wss://libwebsockets.org""
```
Comment (Faless) [inline tutorials/networking/websocket.rst:40]:
```suggestion
    export var websocket_url = "wss://libwebsockets.org"
```

**Comment by Faless** [comment]
Looking good :+1: . Thanks!
You will need to squash the commits together before we can merge. See: https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#the-interactive-rebase

**Comment by Calinou** [comment]
> You will need to squash the commits together before we can merge. See: [docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#the-interactive-rebase](https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#the-interactive-rebase)

We can use GitHub's *Squash and merge* functionality on repositories other than godotengine/godot, so the author doesn't need to do anything here.

However, in the future, please open pull requests on the `master` branch first, or if it's not relevant for `master`, the `3.4` branch (`stable` is just a copy of `3.4`).

**Comment by Calinou** [comment]
Cherry-picked to the `3.5` and `stable` branches. (This appears to already be in `master`.)

**Pull request merged**